doc/cprover-manual/contracts-assigns.md

```diff
@@ -0,0 +1,224 @@
+# Assigns Clause
+
+### Definition
+
```

Comment:
"Definition" is very broad -- how about saying "Syntax"? You've got "Semantics" later.

Reply:
Implemented as suggested.